### PR TITLE
Physx Debugger Content Updates

### LINKED ISSUE #1500: [ISSUE][DOCS] Multiple content issues with the Debugging PhysX documentation page
## Describe the issue

There are multiple issues with the contents of the Debugging PhysX documentation page:
1. "Toggles a visual culling box frame." should be placed under the bolded "Example" to remain consistent with other headers, seen here:
DebuggingPhysX1

2. `physx_CullingBox 1` Console command should not contain a `1` argument (the command should be `physx_CullingBox`), as it is a toggle command, as described above "Toggles a visual culling box frame." (please refer to the https://github.com/o3de/o3de/issues/8455).

3. "3. Click PhysX Debug. Example" in the Debugging with the ImGui Tool section should be either split up to separate lines, or the "Example" text should be deleted, as it looks as a part of the inline text in the point.

4. Debugging with the ImGui Tool section seems to be missing some content below "1. You can make the following changes.", and there are two text separators below it, seen here:
DebuggingPhysX2

## Impacted page(s)

https://www.o3de.org/docs/user-guide/interactivity/physics/debugging/

## Expected experience

Text formatting in the documentation is consistent, and there is no missing content.

### PR DESCRIPTION
**Pull request by madeleinesinging**
## Change summary
Closes #1500 
Signed-off-by Madeleine <jmadeson@amazon.com>

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by chanmosq** [comment]
@madeleinesinging The proper syntax is `Signed-off-by: $NAME <$EMAIL>`, where $NAME and $EMAIL match the configurations in the tool from you're creating the commit. For example, if the tool you're using is a command prompt, run `git config user.name` and `git config user.email` to see the correct name and email.

**Comment by micronAMZN** [comment]
These changes are also included in #1720. I am closing this PR as a duplicate and to avoid merge conflicts.

**Comment by micronAMZN** [comment]
Will close #1720 instead.

File: content/docs/user-guide/interactivity/physics/debugging.md [hunk 1]
```diff
@@ -77,17 +77,23 @@ You must enable the [ImGui Gem](/docs/user-guide/gems/reference/debug/imgui) to
...
+| **Culling** | You can specify the following options:<ul><li>**Wireframe** – Displays wireframes in the viewport.</li><li>**Size** – Click and drag the slider to specify the size of the wireframes. As a best practice, keep this value small to prevent performance issues.</li></ul> |
+| **Collisions** | Enables debugging for collision types. You can specify the following options: <ul><li>**Shapes**</li><li>**Edges**</li><li>**F Normals**</li><li>**Aabbs**</li><li>**Axis**</li><li>**Compounds**</li><li>**Static**</li><li>**Dynamic**</li></ul> |
+| **Body** | Enables debugging for body types. You can specify the following options: <ul><li>**Axes**</li><li>**Mass Axes**</li><li>**Linear Velocity**</li><li>**Angular Velocity**</li></ul> |
 
 ## Debug Options in the PhysX Configuration 
 
-You can also specify debug settings in the **PhysX Configuration** tool. See [Debugger Configuration](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-debugger/).
+You can also specify debug settings in the **PhysX Configuration** tool. See [Debugger Configuration](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-debugger/).
```
Comment (micronAMZN) [inline content/docs/user-guide/interactivity/physics/debugging.md:96]: Quick fix for an extra space.

```suggestion
You can also specify debug settings in the **PhysX Configuration** tool. See [Debugger Configuration](/docs/user-guide/interactivity/physics/nvidia-physx/configuring/configuration-debugger/). 
```

**Review by micronAMZN — APPROVED** [review]
Approved with a very minor fix.

**Review by FiniteStateGit — APPROVED** [review]

**Pull request merged**